plugins/output-elasticsearch/src/main/scala/com/stratio/sparkta/plugin/output/elasticsearch/dao/ElasticSearchDAO.scala

```diff
@@ -22,11 +22,24 @@ import java.util.Date
 
 import org.joda.time.DateTime
 
+import com.stratio.sparkta.sdk.TypeOp
 import com.stratio.sparkta.sdk.TypeOp._
-import com.stratio.sparkta.sdk.{Output, TypeOp}
+import com.stratio.sparkta.sdk._
 
 trait ElasticSearchDAO extends Closeable {
 
+  final val TIMESTAMP_PATTERN = "@timestamp:"
+  final val DEFAULT_DATE_FORMAT = "YYYY.MM.dd"
+  final val YEAR = "YYYY"
+  final val MONTH = "MM"
+  final val DAY = "dd"
+  final val HOUR = "HH"
+  final val MINUTE = "mm"
+  final val SECOND = "ss"
+  final val DEFAULT_INDEX_TYPE = "sparkta"
+  final val DEFAULT_NODE = "localhost"
```

Comment:
This is a bad idea. We doesn't want a default config in code. Moreover, this config only works in local machine!